target_decisioning_engine/utils.py

```diff
@@ -46,33 +38,41 @@ def get_rule_key(rule):
     return rule.get("ruleKey")
 
 
-def _get_default_parsed_url(url):
-    """Returns default dict for urls that cannot be parsed"""
-    parsed = dict(DEFAULT_PARSED_URL)
-    parsed["url"] = url
```

Comment:
What was the reason for getting rid of this particular line?

Reply:
Due to the the DEFAULT_PARSED_URL only being used once after the parsing code change, it felt overkill to have it defined outside of the function. I also removed the helper function for similar reasons.